.github/workflows/test-e2e-semgrep-ci.yml

```diff
@@ -125,6 +125,7 @@ jobs:
     - semgrep-ci
     - semgrep-ci-on-pr
     - semgrep-ci-fail-open
+    - semgrep-ci-fail-open-blocking-findings
```

Comment:
@spencerdrak when I ported this workflow to jsonnet this job was not part of the notify_failure needs list. I'm not sure why, but this looks like a mistake.